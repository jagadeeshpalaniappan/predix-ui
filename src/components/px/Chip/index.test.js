import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import PxChip from './';

describe('Chip', () => {
  test('should render', () => {
    const tree = renderer.create(<PxChip>Title</PxChip>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
