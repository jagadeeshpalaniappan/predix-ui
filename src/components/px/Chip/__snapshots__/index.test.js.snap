// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chip should render 1`] = `
.c0 {
  display: inline-block;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 20px;
  padding-left: 10px;
  padding-right: 10px;
  background-color: var(--px-chip-background-color,white);
  border: 1px solid var(--px-chip-border-color,gray);
}

.c0:hover {
  cursor: pointer;
  background-color: var(--px-chip-background-color--hover,rgba(gray,0.2));
  border: 1px solid var(--px-chip-border-color--hover,gray);
}

.c1 {
  line-height: 18px;
  display: inline-block;
  min-width: 0;
  max-width: var(--px-chip-max-width,120px);
  color: var(--px-chip-text-color,gray);
}

<div
  className="c0"
  selected={null}
>
  <div
    className="c1"
  >
    Title
  </div>
</div>
`;
